css/fares.css

```diff
@@ -28,6 +28,57 @@ ul{
     margin: 0;
 }
 
+#fares-top {
+    background: url("../images/fares-cover-photo.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.header-banner {
+    background-color: #5C6E93;
+    color: white;
+    font-size: 2.7rem;
+    font-family: 'Open Sans', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    padding: 2.4rem;
+    padding-left: 20rem;
```

Comment:
Instead of setting the padding as rem and then adjusting it, find a % that works/looks good so that the responsiveness is taken care of & you don't need the media queries